index.js

```diff
@@ -33,7 +34,8 @@ function handleArgs(cmd, args, opts) {
 		stripEof: true,
 		preferLocal: true,
 		encoding: 'utf8',
-		reject: true
+		reject: true,
+		cleanup: false
```

Comment:
We already have other breaking changes in master. So will be a breaking release regardless.